app/src/test/java/com/example/tmdb/ui/features/login/LoginViewModelTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.tmdb.ui.features.login
+
+import app.cash.turbine.test
+import com.example.tmdb.ViewModelTest
+import com.example.tmdb.domain.repository.UserRepository
+import com.example.tmdb.domain.usecases.LoginParams
+import com.example.tmdb.domain.usecases.LoginUseCase
+import com.example.tmdb.navigations.Navigator
+import io.kotest.matchers.shouldBe
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Test
+
+
+@ExperimentalCoroutinesApi
+class LoginViewModelTest : ViewModelTest() {
+    private val navigator: Navigator = mockk()
+    private val userRepository: UserRepository = mockk()
+    private val loginUseCase: LoginUseCase = LoginUseCase(testDispatcher, userRepository)
+    private val viewModel: LoginViewModel = LoginViewModel(navigator, loginUseCase)
+
+    @Test
+    fun `username change each time user types new input`() = runTest {
```

Comment:
You should check with the password change as well.

Reply:
Fixed